src/components/Person.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
```

Comment:
```suggestion
  <section className="person">
```
Класи краще називати з малої літери. З великої пишуться компоненти, тому не дуже правильно давати класу ім'я компонента

Reply:
![image](https://user-images.githubusercontent.com/86204359/145386159-6049ed31-f157-4f91-93d0-ce8b69444cae.png)
ок, дякую) конфьюзін коли ментори одне правило трактують по-різному))